os/fs/smartfs/smartfs.h

```diff
@@ -378,6 +378,8 @@ int smartfs_mount(struct smartfs_mountpt_s *fs, bool writeable);
 
```

Comment:
please insert space inbetween "shallbe" in the commit message.

Reply:
Updated, thanks